helix/views.py

```diff
@@ -336,6 +337,39 @@ def helix_reso_export_xml(request):
 
     return HttpResponse(rendered_xml, content_type='text/xml')
 
+@api_endpoint
+@api_view(['GET'])
+def helix_solar_production(request, pk=None):
+    org_id = request.GET['organization_id']
+    user = request.user
+    property_state = PropertyState.objects.get(pk=pk)
+    lat = None
+    lng = None
+    capacity = 0
+    errors = []
+    if 'Lat' in property_state.extra_data:
+        lat = property_state.extra_data['Lat']
+    else:
+        errors.append('Property has no Lat column defined')
+    if 'Long' in property_state.extra_data:
+        lng = property_state.extra_data['Long']
+    else:
+        errors.append('Property has no Long column defined')
```

Comment:
please use property.latitude and property.longitude instead:
lat = property_state.latitude
lng = property_state.longitude

Reply:
Those both have the value `None`, even when they are defined in `extra_data`